services/qs-spark-service/src/main/java/ai/quantumics/api/adapter/AwsAdapter.java

```diff
@@ -32,6 +33,7 @@
 import com.amazonaws.services.cloudwatch.AmazonCloudWatchClientBuilder;
 import com.amazonaws.services.cloudwatch.model.*;
```

Comment:
mention the packages explicitly. dont use *

Reply:
will be covered in separate PR